styles/standard.mapcss

```diff
@@ -816,10 +816,11 @@ node|z14-[railway=tram_stop]
 /*****************/
 /* owner changes */
 /*****************/
-node|z11-[railway=owner_change]
+node|z11-[railway=border],
```

Comment:
I wrote at [the commit message](https://github.com/rurseekatze/OpenRailwayMap/pull/480/commits/dbf2246c02999f60b158641bb1da646cd00762b2):
> It is intention to use the same icon for both owner changes and borders.
The grey-black icon would be bad visible on the map.